typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -160,9 +165,8 @@ async function main(): Promise<boolean> {
   startMetricsServer(metricsRegister);
   debug('Starting up', { environment });
 
-  // TODO (Rossy) remove getCoreConfigStub and re-enable getEnvironmentConfig
-  // const coreConfig = getEnvironmentConfig(environment);
-  const coreConfig = getCoreConfigStub(environment);
+  const coreConfig = getEnvironmentConfig(environment);
+  // const coreConfig = getCoreConfigStub(environment);
```

Comment:
just wanna make sure this is still intended to be included

Reply:
Yes indeed. I have a feeling it will be useful for testing Kathy locally in the future